### ISSUE TITLE
Spotify adverts get scrobbled - Lastcast 1.2.0

### ISSUE THREAD

**Issue by Hobby-boy** [erik/lastcast#49]
So I'm currently scrobbling Spotify from my laptop to my chromecast audio via my phone, and I've noticed that when I get an advert on spotify, it also scrobbles the advert as "Spotify - Spotify []". I am using Lastcast 1.2.0 from Pip, and Spotify free.

Log from "journalctl -f _COMM=lastcast": 
```
-- Logs begin at Sun 2019-03-17 10:17:01 GMT. --
Mar 19 19:08:24 DietPine lastcast[13763]: Using chromecast: Bedroom speaker
Mar 19 19:08:26 DietPine lastcast[13763]: Using chromecast: Bedroom Cast
Mar 19 19:10:49 DietPine lastcast[13763]: Scrobbling: Christina Perri - Jar of Hearts [lovestrong.]
Mar 19 19:14:45 DietPine lastcast[13763]: Scrobbling: Duffy - Mercy [Rockferry]
Mar 19 19:18:27 DietPine lastcast[13763]: Scrobbling: Katy Perry - Hot N Cold [One Of The Boys]
Mar 19 19:20:55 DietPine lastcast[13763]: Scrobbling: Spotify - Spotify []
Mar 19 19:21:31 DietPine lastcast[13763]: Scrobbling: Spotify - Spotify []
Mar 19 19:23:42 DietPine lastcast[13763]: Scrobbling: Enrique Iglesias - Hero [Escape]
Mar 19 19:27:54 DietPine lastcast[13763]: Scrobbling: Joe McElderry - The Climb [Wide Awake]
Mar 19 19:31:36 DietPine lastcast[13763]: Scrobbling: Gareth Gates - Anyone Of Us (Stupid Mistake) [What My Heart Wants To Say]
```

**Comment by erik** [comment]
Thanks for reporting!

This seems like a pretty easy fix, I'll try to put out a new version before the end of the week.

However, since you're using spotify, you honestly might be better served by [last.fm's native spotify support](https://getsatisfaction.com/lastfm/topics/spotify-scrobbling), instead of using lastcast.

**Comment by erik** [comment]
This should now be fixed in `lastcast 1.2.1`.

Thanks again for the bug report!
